SECURITY.md

```diff
@@ -0,0 +1,34 @@
+# Security Policy
+
+## Supported Versions
+
+Only the latest version is supported.
+[Firefox might use a slightly older version](https://hg.mozilla.org/mozilla-central/file/tip/package.json#l23), which is supported-ish.
+
+## Out Threat Model 
+### What is considered a vulnerability?
+We assume that a project which makes use of this plugin to apply
+a reasonable amount of scrutiny to all patches that are accepted.
+We understand that this is a fuzzy line and the level of expected
+judgement and sensitivity to security issues will vary between
+projects and reviewers. 
```

Comment:
This section name is "What is considered a vulnerability?" but the section content may not be yet providing a clear answer to that question, maybe the section should be renamed to better match the section content (or maybe we could add to the content something that more directly answer that question?)

As an example, in the last section ("Sneaking your *obfuscated* code past this linter") we are referring to this section, and so it looks that if we provide a more direct example about why a bug related to *obfuscated* code isn't considered part of the threat model targeted by this plugin could provide an answer to both (the title of the section and the link to this section from the last one in this file).

Reply:
I've added a couple of examples. let's see what you think